integration/nextjs.mdx

```diff
@@ -366,6 +367,56 @@ export function MyComponent() {
   </Tab>
 </Tabs>
 
+You can also use pass onSignIn and onSignOut callbacks to useUser to trigger actions when the user signs in or out:
+or the useUser hook, for retrieving information about the user in code:
```

Comment:
nitpick: The "or" after the colon above feels slightly clumsy to me, but maybe it's correct.

Reply:
you're right